src/styles/dimensions.ts

```diff
@@ -0,0 +1,9 @@
+export const dimensions = Object.freeze({
+	headerHeight: "64px",
```

Comment:
Seeing raw pixel sizes is kinda gross, but perhaps the HTML world still does this.